recipes/beauty/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.43.0"
+
+
+class BeautyConan(ConanFile):
+    name = "beauty"
+    homepage = "https://github.com/dfleury2/beauty"
+    description = "HTTP Server above Boost.Beast"
+    topics = ("http", "server", "boost.beast")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "CMakeDeps", "CMakeToolchain"
+
+    requires = ("boost/1.79.0",
+                "openssl/1.1.1o")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "clang": "11",
+            "Visual Studio": "16",
+        }
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```

Comment:
```suggestion
    def config_options_self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
```
self.settings values are available during config_options. However, self.options, are only evaluated after that.

Reply:
Done